src/components/Footer.js

```diff
@@ -0,0 +1,200 @@
+import logo from '../assets/logo_transparent.png';
+
+const Footer = () => (
+  <footer className="">
+    <div className="px-4 py-16 mx-auto max-w-screen-xl sm:px-6 lg:px-8">
+      <div className="flex flex-col items-center p-6 bg-[#08ACB4] rounded-lg shadow-lg gap-4 sm:flex-row sm:justify-between">
+        <strong className="text-xl text-white sm:text-xl">Get to know more about Mov Books!</strong>
+        <a
+          className="inline-flex items-center px-8 py-3 text-[#08ACB4] bg-white border border-white rounded-full hover:bg-transparent hover:text-white active:bg-white/90 focus:outline-none focus:ring transition"
+          href="/"
+        >
+          <span className="text-sm font-medium"> Let's Get Started </span>
+          <svg
+            className="w-5 h-5 ml-3"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
```

Comment:
- [ ] Great job with your project and design 👏 . On the `categories` page you should have just a `button` that says `Check Status` as stated in the project requirement 🙏 . You can always go back to work on this project as you like once you're done submitting it for Microverse 😊

Reply:
This change has been done well.